docs/api/meetings.md

```diff
@@ -0,0 +1,636 @@
+                    "email": "pierren@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 35
+                },
+                {
+                    "name":"Kyouko Sakura",
+                    "email": "kyoukos@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 40
+                },               
+                {
+                    "name":"Dmitry Podkolzin",
+                    "email": "dmitryp@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 79
+                },
+                {
+                    "name":"Jessica Smith",
+                    "email": "jessicas@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 133
+                },                    
+                {
+                    "name": "Chris Hawthorne",
+                    "email": "chrish@dahliacorp.com",
+                    "external": true,
+                    "status": "accepted"
+                },
+                {
+                    "name": "Ada Bottas",
+                    "email": "adab@dahliacorp.com",
+                    "external": true,
+                    "status": "pending"
+                },
+                {
+                    "name": "Conrad Lee",
+                    "email": "conradl@agenda.com",
+                    "external": false,
+                    "status": "declined",
+                    "id": 95
+                }
+            ],
+            "name": "Sync up",
+            "description": "A sync up with the Dahlia Corporation to discuss the Agenda implementation.",
+            "createdAt": 1667302643,
+            "updatedAt": 1667302643,
+            "createdBy": 35,
+            "meetingTime": 1667489400,
+            "series": false
+        }
+    ],
+    "page": 1
+}
+```
+
+| Property | Description |
+| -------- | ----------- |
+| `id` | The ID of the meeting. |
+| `source` | An object that contains details about the meeting's location. |
+| `source.name` | The name of the source. |
+| `source.category` | The category of the source. This value can either be `virtual` for online sources or `in-person` for in-person sources (such as meeting rooms). |
+| `source.type` | The type of the source. Possible values include `zoom`, `teams`, `slack`, and `skype`. This value is **only** returned for sources in the `virtual` category. 
+| `source.url` | The URL for the source. This value is **only** returned for sources in the `virtual` category.
+| `source.id` | The ID of the source. |
+| `invitees` | An array of people invited to the meeting. |
+| `invitees.name` | The name of the person invited. |
```

Comment:
This is nested in an array so the dot notation is incorrect. I'm not too certain on a standard to reference any array item as opposed to a specific one, but something like `invitees[*].name` would probably work.

Reply:
Yeah, that's what I wasn't quite sure about as well, which is why I went just for the dot by itself. Let's go with [*].